.gitignore

```diff
@@ -12,3 +12,6 @@ assets/js/config.js
 .Trashes
 Thumbs.db
 ehthumbs.db
+
+#Idea
```

Comment:
Please don't modify the project's `.gitignore` by adding editor/environment specific files. Those belong in your global `.gitignore`. I think this has been violated a little bit above but we shouldn't make it worse.